docs/macios/xcsync.md

```diff
@@ -20,68 +26,106 @@ Supported file types include:
-    /p:xcSyncXcodeFolder=<TARGET_XCODE_DIRECTORY>
-    /p:xcSyncTargetFrameworkMoniker=<FRAMEWORK>
-    /p:xcSyncVerbosity=<LEVEL>
+dotnet tool install -g dotnet-xcsync
 ```
 
-### xcsync-sync
+## Usage
+
+1. The [.NET MAUI extension](https://aka.ms/mauidevkit-marketplace) provides support for xcsync. Ensure you have the latest version. The xcsync commands in the .NET MAUI extension are only available when using VS Code on macOS.
+2. Open the command palette (Cmd + Shift + P) and search for the `.NET MAUI: xcsync - Generate an Xcode project` and `.NET MAUI: xcsync - Sync from an Xcode project` commands.
+
+## Troubleshooting
+
+To see details on errors related to xcsync, navigate to the **Output** window (<kbd>CMD + Shift + u </kbd>) and select **.NET MAUI - xcsync** in the dropdown. If you are facing issues, please [report an issue](#provide-feedback).
+
+## Provide feedback
+
+Please check existing [Known Issues](https://github.com/dotnet/xcsync/issues) before filing a new issue or suggestion.
+
+<!-- markdownlint-disable MD025 -->
+# [CLI](#tab/cli)
+<!-- markdownlint-enable MD025 -->
+
+## Installation
+
+Install .NET xcsync globally with the following command:
 
 ```dotnetcli
-dotnet build /t:xcsync-sync
-    /p:xcSyncProjectFile=<PROJECT>
-    /p:xcSyncXcodeFolder=<TARGET_XCODE_DIRECTORY>
-    /p:xcSyncTargetFrameworkMoniker=<FRAMEWORK>
-    /p:xcSyncVerbosity=<LEVEL>
+dotnet tool install -g dotnet-xcsync
 ```
 
-### Arguments
+## Synopsis
 
-- **`/p:xcSyncProjectFile=<PROJECT>`**
+```dotnetcli
+xcsync generate [-p|--project <PROJECT>]
+[-tfm|--target-framework-moniker <FRAMEWORK>]
+[-t|--target <TARGET_XCODE_DIRECTORY>] [-f|--force]
+[-o|--open] [-v|--verbosity <LEVEL>]
+[-d|--dotnet-path <PATH>]
 
-    The project file to build. Supported project types are .NET MAUI projects or any .NET project that targets a supported platform (iOS, tvOS, macOS, MacCatalyst).
+xcsync sync [-p|--project <PROJECT>]
+[-tfm|--target-framework-moniker <FRAMEWORK>]
+[-t|--target <TARGET_XCODE_DIRECTORY>] [-v|--verbosity <LEVEL>]
+
+xcsync -h|--help
+```
 
 ### Options
 
-The following options allow you to change the default behavior of the tool:
+- **`-p|--project <PROJECT>`**
 
-- **`/p:xcSyncTargetFrameworkMoniker=<FRAMEWORK>`**
+    The project file to build. Supported project types are .NET MAUI projects or any .NET project that targets a supported platform (iOS, tvOS, macOS, MacCatalyst).
+- **`-tfm|--target-framework-moniker <FRAMEWORK>`**
 
     Invoke the tool for a specific framework. The framework must be defined in the project file. Examples: `net9.0-ios`, `net9.0-maccatalyst`. **Required** if the .NET project supports multiple target frameworks (for example, a standard MAUI project). If a single platform project, the default value will be the single target framework specified in the project file.
+- **`-t|--target <TARGET_XCODE_DIRECTORY>`**
+
+    The directory in which to place the generated Xcode project. Default value is `./obj/xcode`.
```

Comment:
Is this default value still correct? Or is it now $(IntermediateOutputPath)_/xcsync

Reply:
thank you for catching that, yes that should be updated as well!